external-crates/move/crates/move-docgen/src/docgen.rs

```diff
@@ -584,6 +584,15 @@ impl<'env> Docgen<'env> {
             }
         }
 
+        if !module_env.get_enums().count() > 0 {
+            for s in module_env
+                .get_enums()
+                .sorted_by(|a, b| Ord::cmp(&a.get_loc(), &b.get_loc()))
+            {
+                self.gen_enum(&s);
+            }
+        }
```

Comment:
It's weird to me that this checks the count instead of just doing the iteration, but the rest of the block is written that way, too.